src/kontrol/__main__.py

```diff
@@ -733,6 +735,13 @@ def _parse_test_version_tuple(value: str) -> tuple[str, int | None]:
     prove_args.add_argument(
         '--use-gas', dest='use_gas', default=False, action='store_true', help='Enables gas computation in KEVM.'
     )
+    prove_args.add_argument(
```

Comment:
Can you rename this to `--break-on-cheatcodes` instead?

Reply:
Done.